webpack.config.js

```diff
@@ -27,15 +29,25 @@ const extensionConfig = {
     extension: './src/extension.ts'
   },
   resolve: {
-    extensions: ['.js', '.ts', '.json']
+    extensions: ['.js', '.ts', '.json'],
+    fallback: {
+      stream: require.resolve('stream-browserify')
+    }
   },
   externals: {
     // The vscode-module is created on-the-fly and must be excluded.
-    vscode: 'commonjs vscode',
+    vscode: 'commonjs2 vscode',
     // Currently connection-model has a keytar dependency, vscode provides its
     // own keytar dependency. Here we are telling it to use vscode's keytar.
     keytar: 'keytar',
-    electron: 'electron'
+    electron: 'electron',
+    snappy: 'commonjs2 snappy',
+    'snappy/package.json': 'commonjs2 snappy/package.json',
+    'bson-ext': 'commonjs2 bson-ext',
+    'win-export-certificate-and-key': 'commonjs2 win-export-certificate-and-key',
+    'os_dns_native': 'commonjs2 os_dns_native',
+    'mongodb-client-encryption': 'commonjs2 mongodb-client-encryption',
+    encoding: 'utf-8'
```

Comment:
This looks a bit out of place: [externals](https://webpack.js.org/configuration/externals/) in this format will define an external package `encoding` to be resolved as global var with the name `utf-8`